packages/ui/src/Common.ts

```diff
@@ -1529,44 +1524,26 @@ export interface ErrorPageProps {
   resetError: () => void;
 }
 
-export interface FieldProps {
-  name?: string;
-  label?: string;
-  height?: number;
-  type?:
-    | "address"
-    | "boolean"
-    | "currency"
-    | "customSelect"
-    | "date"
-    | "datetime"
-    | "email"
-    | "multiselect"
-    | "number"
-    | "password"
-    | "percent"
-    | "phoneNumber"
-    | "select"
-    | "signature"
-    | "text"
-    | "textarea"
-    | "time"
-    | "url";
-  rows?: number;
-  value?: any;
-  onChange?: any;
-  onBlur?: any;
-  onStart?: any;
-  onEnd?: any;
-  options?: FieldOptions;
-  placeholder?: string;
-  disabled?: boolean;
-  useCheckbox?: boolean;
-  includeCounty?: boolean;
-  googleMapsApiKey?: string;
-  googlePlacesMobileStyles?: Styles;
-  transformValue?: TransformValueOptions;
-}
+export type FieldProps =
+  | TextFieldProps
+  | NumberFieldProps
+  | NumberRangeFieldProps
+  | DateTimeFieldProps
+  | (MultiselectFieldProps & {type: "multiselect"})
```

Comment:
Can this type not be enforced in the interface definition?

Reply:
Do you mean making type="multiselect" on the interface? I went this route so you could do <MultiSelectField ... /> and not have to provide type="multiselect" each time, but you would have to provide it to <Field /> so it knows what to do.

Reply:
Ahhh that makes sense